libraries/chain/include/eosio/chain/hotstuff/hotstuff.hpp

```diff
@@ -160,16 +162,7 @@ namespace eosio::chain {
 
       void reset(const fc::sha256& proposal_id, const digest_type& proposal_digest, size_t num_finalizers, size_t quorum);
```

Comment:
`reset(), is_quorum_met(), num_weak(), num_strong()` all need mutex protection. Careful as `num_weak()` and `num_strong()` are currently already called holding a mutex.

Reply:
Thanks!